pkg/config/config.go

```diff
@@ -315,6 +325,8 @@ type Config struct {
 			Code string `mapstructure:"code"`
 		} `mapstructure:"snippets"`
 	} `mapstructure:"ui"`
+	Diff    map[string]DiffProps   `mapstructure:"diff"`
+	Plugins map[string]PluginProps `mapstructure:"plugins"`
```

Comment:
A reminder to update the docs.

Reply:
yep..
noted [here](https://github.com/treeverse/lakeFS/pull/5387#pullrequestreview-1328934530)